homeassistant/components/hassio/strings.json

```
@@ -22,7 +22,7 @@
       "fix_flow": {
         "step": {
           "system_rename_data_disk": {
-            "description": "'{reference}' is a filesystem with the name 'hassos-data' and is not the active data disk. This can cause Home Assistant to choose the wrong data disk at system reboot.\n\nUse the fix option to rename the filesystem to prevent this. Alternatively you can move the data disk to the drive (overwriting its contents) or remove the drive from the system."
+            "description": "{reference} is a filesystem with the name hassos-data and is not the active data disk. This can cause Home Assistant to choose the wrong data disk at system reboot.\n\nUse the fix option to rename the filesystem to prevent this. Alternatively you can move the data disk to the drive (overwriting its contents) or remove the drive from the system."
```

```suggestion
            "description": "`{reference}` is a filesystem with the name hassos-data and is not the active data disk. This can cause Home Assistant to choose the wrong data disk at system reboot.\n\nUse the fix option to rename the filesystem to prevent this. Alternatively you can move the data disk to the drive (overwriting its contents) or remove the drive from the system."
```
Back ticks should still work

---

Ah ok. I think its fine without either though tbh. It's going to look like `/dev/sda` or `/dev/sdb1` so I think it'll be pretty clear what the name is. There won't be spaces or anything. I was just being overly cautious with the extra quotes.

---

Sure, but as you did in this message, I _think_ we generally should show filepaths in backticks.